### ISSUE TITLE
mixed-indents warns about document with no indents

### ISSUE THREAD

**Issue by jyasskin** [speced/bikeshed#2985]
The following Bikeshed document complains

> WARNING: \`Complain About: mixed-indents yes` is active, but I couldn't infer the document's indentation. Be more consistent, or turn this lint off.

Deleting or merging any of the lines removes the warning.

```bikeshed
<pre class='metadata'>
Title: Title
Shortname: shortname
Level: none
Status: NOTE-ED
Group: tag
Repository: org/repo
URL: https://org.github.io/repo
Editor: An Editor
Abstract: Hello
</pre>


## Heading {#heading1}

Each,
not.
A,
which


### Heading {#heading2}

I
That
the
for
and
as
to
and

Users
for
or
Users
for
Even
user
and
when


### Heading {#heading3}

A
in
These
so
instead


### Heading {#heading4}

A
while
as

A
make
and
```

**Comment by tabatkins** [comment]
Yeah, it's just long enough to trigger the lint (minimum length of 50 lines, I think).  I should put in a bypass for zero indententation at all, the lint is clearly silly on this document.

Or really, probably I should do the "50 lines" count only of indented lines.

**Comment by jyasskin** [comment]
2 indented lines should be enough? I wouldn't want to suddenly notice existing errors just because I added a line.

**Comment by ynyhxfo** [comment]
Actually I also encounter issues on this warning. [Here](https://github.com/WICG/paymentlink/blob/0b66f8096429e052849fe5be1f980bbf9e1c4c2d/index.bs) is my index.bs file, and I only have three consecutive usage of indents. I tried to use tab/space/no indent, and non of them works. Should I add `mixed-indents no` to bypass it?

**Comment by tabatkins** [comment]
> 2 indented lines should be enough? I wouldn't want to suddenly notice existing errors just because I added a line.

I prefer having enough lines to be sure (especially for indent-size inference). I've lowered the limit to 20 lines, tho.

I also just dropped the offending check entirely; it wasn't doing anything useful.
